test/OpenTelemetry.Exporter.Zipkin.Tests/Implementation/ZipkinActivityConversionTest.cs

```diff
@@ -125,5 +126,110 @@ public void ToZipkinSpan_Status_ErrorFlagTest(StatusCode expectedStatusCode, str
                 Assert.DoesNotContain(zipkinSpan.Tags, t => t.Key == "error");
             }
         }
+
+        [Theory]
+        [InlineData(ActivityStatusCode.Unset)]
+        [InlineData(ActivityStatusCode.Ok)]
+        [InlineData(ActivityStatusCode.Error)]
+        public void ToZipkinSpan_Activity_Status_And_StatusDescription_is_Set(ActivityStatusCode expectedStatusCode)
+        {
+            // Arrange.
+            const string description = "Description when ActivityStatusCode is Error.";
+            var activity = ZipkinExporterTests.CreateTestActivity();
+            activity.SetStatus(expectedStatusCode, description);
+
+            // Act.
+            var zipkinSpan = activity.ToZipkinSpan(DefaultZipkinEndpoint);
+
+            // Assert.
+            if (expectedStatusCode == ActivityStatusCode.Unset)
+            {
+                Assert.DoesNotContain(zipkinSpan.Tags, t => t.Key == SpanAttributeConstants.StatusCodeKey);
+            }
+            else
+            {
+                Assert.Equal(
+                    StatusHelper.GetTagValueForActivityStatusCode(expectedStatusCode),
+                    zipkinSpan.Tags.FirstOrDefault(t => t.Key == SpanAttributeConstants.StatusCodeKey).Value);
+            }
+
+            if (expectedStatusCode == ActivityStatusCode.Error)
+            {
+                Assert.Contains(
+                    zipkinSpan.Tags, t =>
+                    t.Key == ZipkinActivityConversionExtensions.ZipkinErrorFlagTagName &&
+                    (string)t.Value == description);
+            }
+            else
+            {
+                Assert.DoesNotContain(
+                    zipkinSpan.Tags, t =>
+                    t.Key == ZipkinActivityConversionExtensions.ZipkinErrorFlagTagName);
+            }
+        }
+
+        [Fact]
+        public void ActivityStatus_Takes_precedence_Over_Status_Tags_ActivityStatusCodeIsOk()
+        {
+            // Arrange.
+            var activity = ZipkinExporterTests.CreateTestActivity();
+            activity.SetStatus(ActivityStatusCode.Ok);
```

Comment:
add a test which does the same thing, but in reverse order. We want to make sure `activity.SetStatus(ActivityStatusCode.Ok);` wins no matter what order they were used.